.github/workflows/deploy.yml

```diff
@@ -5,6 +5,7 @@ on:
 
 env:
   CIBW_BUILD: 'cp37-* cp38-* cp39-*'
+  CIBW_SKIP: "*-win32 *-manylinux_i686"
```

Comment:
Why do we skip these? :thinking:

Reply:
Do we need 32-bit wheels? If so I can pull this out (they were necessary on my local tests, but may not be required on the CI itself).